Documentation/VSTestIntegration.md

```diff
@@ -1,11 +1,13 @@
 # Coverlet Integration with VSTest
```

Comment:
I think we should specify minimum SDK supported version https://github.com/tonerdo/coverlet/issues/395#issuecomment-494666202 2.2.300, right @vagisha-nidhi?

Reply:
And maybe we'll add "simple" documentation like tool and msbuild for users, this is a design doc.

Reply:
Yes. minimum SDK supported version should be specified.